src/config/config.ts

```diff
@@ -47,6 +47,14 @@ export const appConfig = {
       "https://digital-research-books-beta.nypl.org/about?source=catalog",
     circulatingCatalog: "https://nypl.na2.iiivega.com/",
     legacyCatalog: "https://legacycatalog.nypl.org/",
+    loginUrl: {
+      development: "https://dev-login.nypl.org/auth/login",
+      production: "https://dev-login.nypl.org/auth/login",
```

Comment:
let's put the actual production login url here?

Reply:
Yes, this was temporary because we didn't have the env vars down correctly and is waiting PR #99 to be merged in.

Reply:
Updated for clarity but ^